src/main/resources/application.properties

```diff
@@ -4,5 +4,9 @@ spring.datasource.username=root
 spring.datasource.password=password
 dataSource.maximumPoolSize=10
 dataSource.poolName=springHikariCP
+spring.jpa.generate-ddl=true
 spring.h2.console.enabled=true
-logging.level.org.hibernate.SQL=DEBUG
\ No newline at end of file
+logging.level.org.hibernate.SQL=DEBUG
+# App properties
+token.lifetime.ms=100000000
+token.secret=B374A26A71490437AA024E4FADD5B497FDFF1A8EA6FF12F6FB65AF2720B59CCF
```

Comment:
Not a fix requirement, but a hint, read about secrets management and properties encryption